src/main/java/uk/gov/register/views/HomePageView.java

```diff
@@ -69,4 +76,13 @@ public String getLinkToRegisterRegister() {
                 getRegisterId()
         );
     }
+
+    public String getRegisterJson() throws JsonProcessingException {
+        ObjectMapper objectMapper = new ObjectMapper();
+        objectMapper.enable(SerializationFeature.INDENT_OUTPUT);
```

Comment:
does it make sense to (at some point) add this ObjectMapper to the hk2 DI container?